java/src/main/c++/vw_learner_VWLearners.cc

```diff
@@ -48,3 +61,74 @@ JNIEXPORT jobject JNICALL Java_vw_learner_VWLearners_getReturnType(JNIEnv *env,
   return env->GetStaticObjectField(clVWReturnType, field);
 }
 
+// Here are all the prediction functions we might want to use.  Add to this as more predictions are required.
+jint multiclass_predictor(example* vec, JNIEnv *env) { return vec->pred.multiclass; }
```

Comment:
@eisber is looking into creating a tagged union for label and prediction types, effectively allowing dynamic typing. Plausibly, this will be very helpful here.  (More tomorrow, traveling today.)